src/Features/Core/Portable/ConvertTupleToStruct/AbstractConvertTupleToStructCodeRefactoringProvider.cs

```diff
@@ -75,15 +68,15 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             // If it does, we can't convert this.  There is no way to describe this anonymous type
             // in the concrete type we create.
             var fields = tupleType.TupleElements;
-            var containsAnonymousType = fields.Any(p => p.Type.ContainsAnonymousType());
+            var containsAnonymousType = fields.Any<IFieldSymbol>(p => p.Type.ContainsAnonymousType());
```

Comment:
Just curious, why making type parameters explicit? Is it a new style we are using?

Reply:
nope.  it's a fallout of some of our tools not simplifying properly.  i think it's related to the NRT work that was done.  it's a bug :(  i can fix in follwup PR.

Reply:
handled in followup PR.